### DISCUSSION TITLE
Radeon ProRender for Maya 3.4.0

### DISCUSSION THREAD

**Discussion by bsavery** [GPUOpen-LibrariesAndSDKs/RadeonProRenderMayaPlugin#347] [Announcements]
- Support for Maya 2023 has been added.
- The RPR Interactive mode has been improved with the following developments:
  - Better support for more shader node types;
  - Even faster rendering with optimized settings out of the box;
  - Hair rendering support;
  - More AOVs supported;
  - Support for OpenColorIO texture spaces;
  - Shadow Catcher support.
- A new Bevel Shader node has been added. This can be used to create small rounded edges on objects. In the physical world, objects rarely have completely sharp edges like polygons. Using this node will enhance the details of artists’ renders.
- Support for the Double Sided node has been added, which allows attaching a different shader to each side of an object mesh.
- A new RPR Ramp node type has been added.
- The “Pixel Filter” (anti-aliasing) setting for RPR Final has been re-enabled.

<hr /><em>This discussion was created from the release <a href='https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderMayaPlugin/releases/tag/'>Radeon ProRender for Maya 3.4.0</a>.</em>

**Comment by actavi01** [comment]
kindly sir fix it ..the camera animation does not work.and does not saved that particular render in render sequence

**Reply by bsavery** [reply]
Can you file an issue with a scene file and explain what doesn't work?

**Comment by actavi01** [comment]
![Screenshot (9)](https://user-images.githubusercontent.com/118048059/201794192-e3efe8f2-bb56-4796-8594-0443421e802b.png)
here is the scene sir,,,it does not get rendered that specified folder.if i switched to one single frame render then i can able to save in it.and minor bug i just noticed when i panning around or if zoom in/out the the prospective switched to original position .

**Comment by actavi01** [comment]
![Screenshot (10)](https://user-images.githubusercontent.com/118048059/201794671-5b33cec9-d359-46d1-b5a6-6185fb96cd95.png)
my specified folder where the image supposed to be rendered.

**Comment by actavi01** [comment]
![Screenshot (5)](https://user-images.githubusercontent.com/118048059/201794794-a6c74005-6fd8-46f7-97f7-7b3bc5aa82c9.png)
![Screenshot (6)](https://user-images.githubusercontent.com/118048059/201794802-2832a451-c5e6-48d9-a43e-b104915f8582.png)
![Screenshot (7)](https://user-images.githubusercontent.com/118048059/201794806-e81bd1c8-8f51-443f-b151-cd37bb299230.png)
![Screenshot (4)](https://user-images.githubusercontent.com/118048059/201794812-2bdd0434-7415-46b6-a5fe-1e107f438a34.png)
![Screenshot (1)](https://user-images.githubusercontent.com/118048059/201794817-16e747c6-ebe9-409c-b49a-94cef11440b4.png)
here specs my pc including default arnold render
